.buildbot.sh

```diff
@@ -4,10 +4,16 @@
 
 set -e
 
+case ${STD_TRACER_MODE} in
+    "sw" | "hw") true;;
+    *) echo "STD_TRACER_MODE must be set to either 'hw' or 'sw'"
+        exit 1;;
```

Comment:
Minor inconsistent indent?

Reply:
Yeah, I was unsure how to format this. How would you have done it?

Reply:
I would line the `exit` up to `echo` myself.